spanner/spannertest/integration_test.go

```diff
@@ -398,6 +398,55 @@ func TestIntegration_SpannerBasics(t *testing.T) {
 	if !reflect.DeepEqual(ages, wantAges) {
 		t.Errorf("Query with IN UNNEST gave wrong ages: got %+v, want %+v", ages, wantAges)
 	}
+
+	// Verify boolean query parameters
```

Comment:
please revert the `integration_test.go` changes. the other tests are sufficient.